eng/Versions.props

```diff
@@ -8,6 +8,7 @@
     <PatchVersion>0</PatchVersion>
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
     <PreReleaseVersionIteration>2</PreReleaseVersionIteration>
+    <PreReleaseVersionIteration Condition="$(MSBuildProjectName.StartsWith('Microsoft.Extensions.'))">$(PreReleaseVersionIteration)-runtime</PreReleaseVersionIteration>
```

Comment:
@JunTaoLuo here's the workaround we're going to use temporarily to permit package overlap between extensions and runtime.

Reply:
Sounds good